docs/conf.py

```diff
@@ -53,6 +53,17 @@
 html_theme = 'pandas_sphinx_theme'
 html_logo = '_static/pandas.svg'
 
+html_theme_options = {
+    "external_links": [
+        {
+            'url': "https://pandas.pydata.org/pandas-docs/stable/",
```

Comment:
Could also be a tuples of (name, url), but the dict is also good for me (and a dict is more easily extendable, if needed in the future).

Reply:
And I suppose the dict is also nicer to use in the jinja layout

Reply:
yeah I was thinking a dict because it'll be more obvious in the templates